### DISCUSSION TITLE
Q: How to encrypt disk image (.qcow2) Windows 11 ARM

### DISCUSSION THREAD

**Discussion by tommyboylab** [utmapp/UTM#5033] [UTM Help (macOS)]
I'm trying to encrypt the disk image generated from UTM but have not found any discussion regarding the process.

I've tried using both the built-in disk encryption in Windows 11 as well as Bitlocker, but both give me an error when trying to use them.

I'd rather encrypt the base image locally but am not sure if having it hosted in a mac DMG I mount then open the VM would be viable.

Would someone help me with the process of encryption?

UTM's tagline is `Securely run operating systems on your mac` so there must be a way I assume.

**Comment by dltacube** [comment]
Is this really not possible? You ever find a solution to your answer?
